documentation/source/ObjectDetection.md

```diff
@@ -310,9 +309,7 @@ class MyNewDetectionDataset(DetectionDataset):
             lines = targets_file.read().splitlines()
             target = np.array([x.strip().strip(',').split(',') for x in lines], dtype=np.float32)
 
-        res_target = np.zeros((self.max_targets, 5))  # cls, cx, cy, w, h
```

Comment:
Same here, if `max_targets` passed explicitly - the old functionality should be preserved for now.

Reply:
I'll update transforms with the warning, asking about the notebook

Reply:
Code examples in notebooks & docs are same-class citizens as regular code. To some extent they are even more important, since this is what users will see first when searching.
So yes, they should reflect up-to-date, recommended way of using SG.